internal/pkg/template/templates/environment/partials/environment-manager-role.yml

```diff
@@ -234,6 +234,13 @@ EnvironmentManagerRole:
             "kms:Decrypt"
           ]
           Resource: "*"
+{{- if .PublicHTTPConfig.ELBAccessLogs.ShouldCreateBucket}}
+        - Sid: EmptyEnvManagedBucket
```

Comment:
nit: because the policy is pretty specific to the access log bucket ⬇️ 
```suggestion
        - Sid: EmptyELBAccessLogsBucket
```

It'd made sense to name it `EmptyEnvManagedBucket` if this policy applies on resources tagged with `copilot-application` and `copilot-environment`.